src/main/java/org/fcrepo/spec/testsuite/crud/HttpPut.java

```diff
@@ -112,8 +112,10 @@ public void httpPutUpdateTriples() {
                                         SPEC_BASE_URL + "#http-put-ldprs", ps);
         final Response resource = createBasicContainer(uri, info);
         final String locationHeader = getLocation(resource);
-
-        final Response getResponse = doGet(locationHeader);
+        final Header preferHeader = new Header("Prefer",
+                "return=representation; include=\"http://www.w3.org/ns/ldp#PreferMinimalContainer\"; " +
+                        "omit=\"http://fedora.info/definitions/v4/repository#ServerManaged\"");
```

Comment:
The new API prefer header for SMTs is `http://fedora.info/definitions/fcrepo#ServerManaged`
https://github.com/fcrepo/fcrepo-specification/commit/aa49ae1af5aabaa3d4b035729049d2eac95e526d#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R382